packages/super-components/_common/mock-helpers.ts

```diff
@@ -0,0 +1,52 @@
+import { apiClient } from '@ovh-ux/manager-core-api';
+import MockAdapter from 'axios-mock-adapter';
+import { Handler } from './msw-helpers';
+
+export const mockRequests = (handlers: Handler[]) => {
+  const mock = new MockAdapter(apiClient.v6);
+  handlers.forEach(({ url, method = 'get', status = 200, response }) => {
+    const mockMethod = `on${method[0].toUpperCase()}${method?.substring(1)}`;
+    const mockUrl =
+      url.includes('*') || url.includes(':')
+        ? new RegExp(url.replace('*', '.*').replace(':id', '*'))
```

Comment:
**suggestion:** `url.replace('*', '.*').replace(':id', '*')` is used twice. Maybe this can be extracted to a function.

Reply:
done thank you